templates/form-element-label.html.twig

```diff
@@ -0,0 +1,12 @@
+{#
+/**
+ * @file
+ * Template override for a form element label.
+ */
+#}
+
+{% extends '@classy/form/form-element-label.html.twig' %}
+
+{%
+  set attributes = attributes.addClass('form-item__label')
```

Comment:
I'm wondering if we shouldn't use the BEM block element pattern here since theoretically this could be used outside of the `form-item` block.

Reply:
If it's outside of the form-item, than its an another element. I'm sure that the `form-element-label.html.twig` will be rendered inside a form-element.

An alternative solution would be to add this at `claro_preprocess_form_element` but that would make no sense imho.

Reply:
@lauriii ^^ ?

Reply:
I think the current solution is fine. If we run into problems with this, let's change it 🙂